tests/cluster_recipes/test_murata_binned_spec_z.py

```diff
@@ -179,7 +179,7 @@ def test_evaluates_theory_prediction_returns_value(
     mock_bin.z_edges = (0, 1)
 
     prediction = murata_binned_spec_z.evaluate_theory_prediction(
-        cluster_abundance, mock_bin, 360**2
+        [cluster_abundance], mock_bin, 360**2
     )
```

Comment:
There is one test failing here because it is trying to obtain the Property not implemented error, which I removed from the code. I did not erased this test because I want to know if we are going to put another warning, as per my comment above